gtsam/slam/tests/testEssentialMatrixWithCalibrationFactor.cpp

```diff
@@ -0,0 +1,446 @@
+/**
+ * @file testEssentialMatrixWithCalibrationFactor.cpp
+ * @brief Test EssentialMatrixWithCalibrationFactor class
+ * @author Ayush Baid
+ * @author Akshay Krishnan
+ * @date April 22, 2021
+ */
+
+#include <gtsam/slam/EssentialMatrixWithCalibrationFactor.h>
```

Comment:
Same comment, I would probably prefer to keep it in the same file and reuse the examples. It’s all two- view geometry so all this copy/paste does not really needed.

Reply:
merged the files